### DISCUSSION TITLE
Can't find location of cursor while erasing

### DISCUSSION THREAD

**Discussion by gerbigtim** [xournalpp/xournalpp#2825] [Q&A] [answered]
Hello,

is there an option to make the eraser have an indicator? Since I'm using a mouse currently I have no way of knowing where the cursor is in case I want to erase something.

Thanks in advance
Tim

**Comment by rolandlo** [comment, marked as answer]
I guess you are using the flatpak. In that case the bug is known, see #2751 and #2375.

**Reply by gerbigtim** [reply]
Ah good to know. Luckily it resolved itself at some point (no idea what i did though).
